cmd/krel/templates/latest/metadata.yaml

```diff
@@ -74,6 +74,8 @@ kubeadm:
     dependencies:
       - name: cri-tools
         versionConstraint: ">= 1.30.0"
+  - versionConstraint: ">= 1.32.0"
+    sourceURLTemplate: "{{ KubernetesURL }}"
```

Comment:
Why doesn't `kubeadm` depend on `kubelet`, `kubectl`, and `kubernetes-cni` any longer?

Reply:
This has been changed in https://github.com/kubernetes/release/pull/3481

Reply:
kubeadm will not install or manage kubelet or kubectl for you, so you will need to ensure they match the version of the Kubernetes control plane you want kubeadm to install for you. If you do not, there is a risk of a version skew occurring that can lead to unexpected, buggy behaviour.

- https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl
-

Reply:
It looks good, but I'm not familiar with the release process. I also want others from kubeadm to review whether it can be merged in this release cycle.

Reply:
@saschagrunert Ah, I forgot about this PR, thanks for pointing it out! 😅